doc/sphinx/changes.rst

```diff
@@ -14,6 +14,100 @@ Version 8.14
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+Coq version 8.14 integrates many usability improvements, as well
+as an important change in the core language. The main changes include:
+
+  - The :ref:`internal representation <814CaseRepresentation>` of `match` has changed
+    to a more space-efficient and cleaner structure, allowing to fix a completeness
+    issue in presence of cumulative inductive types in the type-checker.
+    The internal representation is now closer to the user-level view of `match`,
+    where the argument context of branches and the inductive binders `in` and `as`
+    do not carry type annotations.
+
+  - A :ref:`new <814CoqNative>` `coqnative` binary performs separate native compilation
+    of libraries, starting from a `vo` file, and is supported by `coq_makefile`.
+    :ref:`Linking of native code libraries <814NativeLinking>` is now performed lazily,
+    bringing a gain of performance on some systems.
```

Comment:
Really a gain in performance? cc @ppedrot @ejgallego

Reply:
I think that this entry refers to two different things. The second one has nothing to do with `coqnative`, it's really about linking in the Coq process, and indeed it's a workaround to OSX linking being slow.

Reply:
Oh right, I've read too fast and thought this was a single thing. Maybe should be split in two bullet points then.

Reply:
I've just removed the second point which is minor.